packages/form/addon/components/cf-field/input/radio.js

```diff
@@ -0,0 +1,26 @@
+import { action } from "@ember/object";
+import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
+
+export default class CfFieldInputRadioComponent extends Component {
+  @tracked checkedValue = this.args.field.answer.value;
+
+  @action
+  updatSelection({ target: { value, checked } }) {
+    if (this.args.field.question.raw.isRequired === "false" && checked) {
+      this.checkedValue = value;
+    } else if (this.args.field.question.raw.isRequired === "true") {
+      this.args.onSave(value);
+    } else {
+      this.args.onSave(null);
+    }
+  }
```

Comment:
Why is this method needed? Seems to me that we only need `resetSelected`.

Reply:
`reverted`